app/views/wallaby/resources/index/_link.html.erb

```diff
@@ -0,0 +1,5 @@
+<%# @param object [model] model instance %>
+<%# @param field_name [String] name of the field %>
+<%# @param value [Object] value of the field %>
+<%# @param metadata [Hash] metadata of the field %>
+<%= value.nil? ? null : link_to(metadata[:label], value, metadata[:html_options]) %>
```

Comment:
`metadata[:label]` is normally used as the field name. for instance, if a field is `updated_at` and its value is `2018-08-09`, and `metadata[:label]` is set to `Last Modified` then it will look like this in index page:

```
Last Modified
-------------------
2018-08-09
```

so in your case, it will become

```
View Product
-------------------
View Product
```

maybe metadata[:title] would be a better option for the link title?

Reply:
Happy for it to be changed to `title`